### ISSUE TITLE
Adding HTML into Form lacks sending form (gives ERROR)

### ISSUE THREAD

**Issue by RemPro** [elementor/elementor#11647]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by RemPro** [comment]
When I use HTML option to put extra text between the input fields it gives an error while sending. Without it, it sends perfectly. I did not use special character etc, just plain text.

**Comment by lior-sh** [comment]
Hi @RemPro,

Thanks for reporting, we couldn't reproduce the issue.
Please follow guidelines for opening a new [issue report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md), so we can help you better.
This report is missing key information:
* Prerequisites.
* **Steps to replicate.**
* Isolating the problem.
* **System info**.
* One issue per thread
Without it, we simply can't help.
Thanks.

**Comment by RemPro** [comment]
Hereby I send you the json template that gives the problem.




Groeten



-Rem

i.v.m. de zomervakantie ben ik gesloten van 6 t/m 24 juli


email- en webhosting • domeinregistratie • websites • drukwerk
ontwerp • computer reparatie en installatie • advies en support

Witteweg 9, 1431 GZ  Aalsmeer • telefoon: 06 50535013
e-mail: info@rempro.nl [mailto:info@rempro.nl] • internet: www.rempro.nl [http://www.rempro.nl]

Dit e-mailbericht is alleen bestemd voor de geadresseerden.Indien dit
bericht niet voor u is bedoeld, wordt u verzocht de afzender hiervan op
de hoogte te stellen door het bericht te retourneren en de inhoud niet
te gebruiken. Aan dit bericht kunnen geen rechten worden ontleend.

Op 16-6-2020 17:47:32, Lior <notifications@github.com> schreef:
Hi @RemPro [https://github.com/RemPro],
Thanks for reporting, we couldn't reproduce the issue.
Please follow guidelines for opening a new issue report [https://github.com/pojome/elementor/issues/new?template=Bug_report.md], so we can help you better.
This report is missing key information:
* Prerequisites.
* Steps to replicate.
* Isolating the problem.
* System info.
* One issue per thread
Without it, we simply can't help.
Thanks.

**Comment by shilo-ey** [comment]
@RemPro there is no JSON file attached

**Comment by RemPro** [comment]
Problem Solved with the latest Elementor Pro Version 2.10.2
